tests/debugging/test_debugger.py

```diff
@@ -717,6 +719,11 @@ def test_debugger_condition_eval_then_rate_limit():
         for i in range(100):
             Stuff().instancestuff(i)
 
+        # Not sure this test is valid.
+        # Stuff().instancestuff(42) called only once.
+        # Therefore. it only check that condition executed before rate limit.
+        # If rate limit was not working this doesn't catch it as it only run once.
```

Comment:
The property we're testing here is that the condition is evaluated before rate limit. The rate limiter has its own suite of tests. So I think this should be fine 🙂 .